src/docker/v2/handlers/manifests.rs

```diff
@@ -200,15 +199,8 @@ fn internal_error_response(
 fn sign_and_save_package_version(
```

Comment:
This method is no longer signing:

```suggestion
fn save_package_version(
```